data_pipeline/surveymonkey/surveymonkey_etl.py

```diff
@@ -30,13 +30,71 @@ def get_survey_list(access_token: str) -> list:
 def get_survey_question_details(access_token: str, survey_id: str) -> list:
     headers = get_surveymonkey_api_headers(access_token)
     response = requests.get(
-        f'https://api.surveymonkey.com/v3/surveys/{survey_id}/details',
+        f"https://api.surveymonkey.com/v3/surveys/{survey_id}/details",
         headers=headers
     )
     reponse_json = response.json()
     return reponse_json
 
 
+def get_no_answer_options_for_question_json() -> dict:
+    return {
+        "id": None,
+        "text": None,
+        "type": None,
+        "weight": None
+    }
+
+
+def parse_answer_options_in_question_answer_json(
+    question_answer_response_json: dict,
+    option_key: str
+) -> dict:
+    if isinstance(question_answer_response_json[option_key], list):
+        result_list = []
+        for answer_option in question_answer_response_json[option_key]:
+            result_list.append({
+                "id": answer_option.get("id"),
+                "text": answer_option.get("text"),
+                "type": answer_option.get("type"),
+                "weight": answer_option.get("weight")
+            })
+        return result_list
+    return [{
+        "id": question_answer_response_json[option_key].get("id"),
+        "text": question_answer_response_json[option_key].get("text"),
+        "type": question_answer_response_json[option_key].get("type"),
+        "weight": question_answer_response_json[option_key].get("weight")
+    }]
+
+
+def parse_answers_in_question_json(question_response_json: dict) -> dict:
+    result = {
+        "choices": [get_no_answer_options_for_question_json()],
```

Comment:
I see now where the "no answer" value comes from. I don't know the benefit of having the "empty" option in this case. Maybe it is worth adding a comment regarding why we add it.

Maybe I would suggest to move that logic to `parse_answer_options_in_question_answer_json`.
Then you could potentially also remove the `if` statements before calling `parse_answer_options_in_question_answer_json`?

Reply:
Just adding a comment that this has now changed to use an empty list instead.